eng/Versions.props

```diff
@@ -6,6 +6,7 @@
     <RestoreSources>
       $(RestoreSources);
       https://api.nuget.org/v3/index.json;
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json;
```

Comment:
I'm assuming its's ok to leave this in? That way the next time we need to update MDA, we don't have to search for the feed?

Reply:
I would remove it unless absolutely necessary. :)